cloud/cloud_env_impl.cc

```diff
@@ -851,15 +851,15 @@ Status CloudEnvImpl::LoadLocalCloudManifest(const std::string& dbname) {
   if (cloud_manifest_) {
     cloud_manifest_.reset();
   }
-  return CloudEnvImpl::LoadLocalCloudManifest(dbname, GetBaseEnv(),
-                                              &cloud_manifest_);
+  return CloudEnvImpl::LoadLocalCloudManifest(
+      dbname, GetBaseEnv(), cloud_env_options.cookie_on_open, &cloud_manifest_);
 }
 
 Status CloudEnvImpl::LoadLocalCloudManifest(
-    const std::string& dbname, Env* base_env,
+    const std::string& dbname, Env* base_env, const std::string& cookie,
     std::unique_ptr<CloudManifest>* cloud_manifest) {
   std::unique_ptr<SequentialFileReader> reader;
-  auto cloud_manifest_file_name = CloudManifestFile(dbname);
+  auto cloud_manifest_file_name = MakeCloudManifestFile(dbname, cookie);
```

Comment:
See below. I'm thinking our local cloudmanifest filename could be without the cookie suffix.